### ISSUE TITLE
היערכות למונדיאל?

### ISSUE THREAD

**Issue by rafi-d** [Fishenzon/repo#12]
נראה לי שיש כוונה להעביר שידורים חיים מהמונדיאל לאתר הזה:
https://mondial.kan.org.il/
כדאי להיערך בהתאם....

**Comment by Fishenzon** [comment]
@rafi-d 
אין שם כרגע שום שידור

**Comment by rafi-d** [comment]
מתוכנן שיהיה... :)

**Comment by Fishenzon** [comment]
@rafi-d 
מוזמן לבדוק בגירסה 1.5.5

**Comment by rafi-d** [comment]
יפה!

מעניין אם הם ימשיכו בשני ערוצי אודיו או ישפרו ל5.1
אגב, לפי התכנון, אמורים להיות מספר שידורי משחק(ים) במקביל באתר (זוויות או שני משחקים). יש מצב אולי איכשהו להכניס את זה לתוסף?

**Comment by rafi-d** [comment]
אגב, זה רק נדמה לי או שהערות הרגיל של כאן 11 פתאום מוצג בSD במקום FHD הרגיל?

**Comment by Fishenzon** [comment]
@rafi-d 
לגבי בחירה בשידורים שונים, צריך לראות קודם איך זה יעבוד באתר, זה עדיין לא קיים.
לגבי ערוץ 11, תראה באתר שהם הורידו לו את האיכות.

**Comment by rafi-d** [comment]
נכון, קלקלו את האתר האיכותי שלהם :( אולי אפשר לקחת את הערוץ הזה מהוט או יס במקום?

**Comment by Fishenzon** [comment]
כל השידורים בהרחבה נלקחים ישירות מהאתרים, מה שיש באתרים זה מה שרואים

**Comment by rafi-d** [comment]
כל ספקיות הIPTV מספקות את כאן 11 
בHD
אי שם אולי יש FEED שלהם באינטרטנט...

**Comment by rafi-d** [comment]
נדמה לי שה
EPG
השתבש בגרסאות האחרונות. תוכל לבדוק?

**Comment by Fishenzon** [comment]
@rafi-d 
מה בדיוק השתבש?

**Comment by rafi-d** [comment]
כעת נראה תקין. מה שקרה, לדעתי זה שעבדתי זמן מסוים עם 
VPN
ואז כל הזמנים פתאום קיבלו תזוזה מוזרה קדימה (הוצגו תוכניות לא נכונות לזמנים מאוחרים יותר)
כעת נראה תקין...
